packages/explorer/src/app/(explorer)/[chainName]/worlds/[worldAddress]/explore/useQueryValidator.ts

```diff
@@ -10,26 +10,49 @@ import { useMonacoErrorMarker } from "./useMonacoErrorMarker";
 
 const sqlParser = new Parser();
 
+function findErrorPosition(query: string, erroredQueryStr: string) {
+  const lines = query.split("\n");
+  let startLineNumber = 1;
+  let startColumn = 1;
+  let currentPosition = 0;
+
+  for (let i = 0; i < lines.length; i++) {
+    if (currentPosition + lines[i].length >= query.indexOf(erroredQueryStr)) {
+      startLineNumber = i + 1;
+      startColumn = query.indexOf(erroredQueryStr) - currentPosition + 1;
+      break;
+    }
+    currentPosition += lines[i].length + 1; // +1 for newline character
+  }
+
+  return {
+    startLineNumber,
+    endLineNumber: startLineNumber,
+    startColumn,
+    endColumn: startColumn + erroredQueryStr.length,
```

Comment:
noticed that below we also add 1 to `endColumn` value, do we need this here too?

Reply:
it ends up with +1 being added a few lines above (`startLineNumber = i + 1`). Double-checked that columns are marked correctly just in case